modules/Registration/components/StudentRegistrationForm.tsx

```diff
@@ -0,0 +1,299 @@
+import React from "react";
+
+import { zodResolver } from "@hookform/resolvers/zod";
+import { TextInput, Select, PasswordInput, Button, Box, Group, Stack } from "@mantine/core";
+import { useForm, Controller } from "react-hook-form";
+
+import styles from "./StudentRegistraionForm.module.css";
+import { IStudentRegistrationFormProps } from "./StudentRegistrationForm.type";
+import {
+  StudentRegistrationFormData,
+  studentRegistrationSchema,
+  defaultValues,
+} from "./studentRegistrationSchema";
+
+export function StudentRegistrationForm({ onSubmit }: IStudentRegistrationFormProps) {
+  const {
+    control,
+    register,
+    handleSubmit,
+    watch,
+    formState: { errors },
+    reset,
+  } = useForm<StudentRegistrationFormData>({
+    resolver: zodResolver(studentRegistrationSchema),
```

Comment:
Export the resolver from `StudentRegistrationForm.helpers.ts` file and import it here like so: 
```suggestion
    resolver: StudentRegistrationSchemaResolver,
```